AtlasStarter/Program.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System;
+using System.Collections.Generic;
+using MongoDB.Driver;
+
+namespace AtlasStarter
+{
+    class MainClassNoPrompt
+    {
+        public static void Main(string[] args)
+        {
+            // Provide your Atlas connection string here. Be sure it includes
+            // a valid username and password! Note that in a production environment,
+            // you do not want to store your password in plain-text here.
+
+            var mongoUri = "";
```

Comment:
a TODO above this might help as well!

Reply:
Providing (either in comments or in the string itself) a sample URI might make it easier for new developers to get used to the format: https://github.com/nathan-contino-mongo/java-atlas-example/blob/master/src/main/java/mongodb/Main.java#L21